fastlane/lib/fastlane/actions/mailgun.rb

```diff
@@ -114,6 +119,13 @@ def self.mailgunit(options)
         unless options[:reply_to].nil?
           params.store(:"h:Reply-To", options[:reply_to])
         end
+
+        unless options[:attachment].nil?
+          attachment_filenames = [*options[:attachment]]
+          attachments = attachment_filenames.map { |filename| File.new(filename, 'rb') }
```

Comment:
I'm not very familiar with the distinction, but this is how it appears in [rest-client's docs](https://github.com/rest-client/rest-client#multipart) so I didn't think too much of it. 

From documentation it seems like the `b` option causes no CRLF conversion to be performed, which is obviously the case for attachments.